src/db_tables/media.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import ForeignKey
```

Comment:
```suggestion
"""This file houses all tvdb media related database tables.

Some of these tables only have one column (`tvdb_id`), which may seem like a mistake, but is intentional.
That's because this provides better type safety and allows us to define proper foreign key relationships that
refer to these tables instead of duplicating that data. It also may become useful if at any point we would
want to store something extra that's global to each movie / show / episode.
"""

from sqlalchemy import ForeignKey
```